package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@rafflebox-technologies-inc/lambda-typescript",
+  "version": "1.0.0",
+  "main": "index.js",
+  "description": "A simple example of a quick start for a new project",
+  "license": "MIT",
+  "author": "Rafflebox",
+  "engines": {
+    "node": "^14 || ^16"
+  },
+  "scripts": {
+    "clean": "rm -rf build/",
+    "build": "yarn clean && tsc",
+    "lint": "eslint \"**/*.{ts,js}\"",
+    "format": "prettier --write \"**/*.{ts,js,json,md,yml}\""
```

Comment:
We should have a script to package the lambda into a `.zip` file for uploading to aws.